src/navigation/post_backup.py

```diff
@@ -13,7 +13,7 @@
 import tf2_ros
 import rospy
 
-POST_RADIUS = get_rosparam("gate/post_radius", 0.7)  * get_rosparam("single_fiducial/post_avoidance_multiplier", 1.42)
+POST_RADIUS = get_rosparam("gate/post_radius", 0.7) * get_rosparam("single_fiducial/post_avoidance_multiplier", 1.42)
 BACKUP_DISTANCE = get_rosparam("recovery/recovery_distance", 2.0)
 STOP_THRESH = 0.2
```

Comment:
I think this PR is missing some updates that were made in the backup at post (#481) PR such as using get_rosparam here and the params and return listed in the docstring for the avoid_post_trajectory function.